instant-gram/ViewControllers/HomeFeedViewController.m

```diff
@@ -0,0 +1,40 @@
+//
+//  HomeFeedViewController.m
+//  instant-gram
+//
+//  Created by Joey R. Hernandez Perez on 7/7/20.
+//  Copyright © 2020 joeyrhernandez1. All rights reserved.
+//
+
+#import "HomeFeedViewController.h"
+#import "LoginViewController.h"
+#import <Parse/Parse.h>
+#import "SceneDelegate.h"
+
+@interface HomeFeedViewController ()
+
+@end
+
+@implementation HomeFeedViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+}
+
+- (IBAction)onTapLogout:(id)sender {
```

Comment:
The timing on this could be weird. I haven't tested it yet but you're doing navigation stuff on the main thread while changing global app state asynchronously. I wonder if you used a super slow connection if you could get yourself into a bad state...